pkg/apps/util/util_test.go

```diff
@@ -586,3 +586,22 @@ func TestRemoveCondition(t *testing.T) {
 		}
 	}
 }
+
+func TestRolloutExceededTimeoutSeconds(t *testing.T) {
```

Comment:
I'd like to see a test table containing setting different strategies and not specified timeouts mixed with creationTimestamps values

Reply:
done.